packages/core/src/components/MonthPicker/MonthPicker.jsx

```diff
@@ -76,7 +78,19 @@ export class MonthPicker extends React.PureComponent {
       } else {
         selectedMonths.push(month);
       }
+
       this.setState({ selectedMonths });
+
+      this.handleAriaPressed(selectedMonths);
```

Comment:
Nit: the `handle...` naming convention is used to indicate that a method is used as an event listener. That's not totally the case here, and I'm wondering if this would be more accurately described as `updatedAriaPressed`?